sampleDriver/EvsStreamingCamera.cpp

```diff
@@ -73,9 +74,53 @@ EvsStreamingCamera::EvsStreamingCamera(const char *deviceName, unique_ptr<Config
               GRALLOC_USAGE_SW_WRITE_OFTEN;
 
     char propValue[PROPERTY_VALUE_MAX] = {};
-    property_get("persist.vendor.evs.uri", propValue, "ws://192.168.122.1:8888");
+    property_get("persist.vendor.evs.uri", propValue, "ws://192.168.122.1");
     mUri = propValue;
-    LOG(DEBUG) << "EvsStreamingCamera instantiated " << mUri;
+
+    if (!std::string(deviceName).compare("CarlaSimFront")) {
+        mUri += ":8887";
```

Comment:
Hi @DKlimenko, should these ports be hard-coded? Or, maybe, it is worth making them separate properties?